getting-started/photometry.rst

```diff
@@ -22,107 +20,87 @@ This is the measurement strategy:
 
-Notice that since this task operates on coadds, we need to select the coadds using the ``filter``, ``tract``, and ``patch`` data ID keys.
+Notice that since this task operates on coadds, we can select the coadds using the ``tract``, and ``patch`` data ID keys.
+In past sections, the examples left off the ``-d`` argument in order to process all available data.
+This example, however, is selecting just four of the patches for this step.
+Those four patches have coverage from all 40 visits in the tutorial repository which means there doesn't need to be as much fine tuning to configurations, and we can process these patches just as the large scale HSC processing is done.
+As with previous examples, the outputs will go in a collection placed under a namespace defined by your username.
 
-Also notice that you've created a new rerun for the photometry outputs, ``coaddPhot``, that is chained to the ``coadd`` rerun.
+.. note:
 
-Now repeat source detection in ``HSC-I``-band patches:
+  The processing in this part can be quite expensive and take a long time.
+  You can use the `-j <NUM>` argument to allow the processing to take more cores, if you have access to more than one.
 
-.. code-block:: bash
+Detecting sources in coadded images
+===================================
 
-   detectCoaddSources.py DATA --rerun coaddPhot \
-       --id filter=HSC-I tract=0 patch=0,0^0,1^0,2^1,0^1,1^1,2^2,0^2,1^2,2
+To start, detect sources in the coadded images to take advantage of their depth and high signal-to-noise ratio.
+The ``detection`` sub-pipeline is responsible for producing calibrated measurements from the input coadds.
+Detection is done on each band and patch separately.
 
-The :command:`detectCoaddSources.py` commands produce ``deepCoadd_det`` datasets in the Butler repository.
-Typically these datasets are only used as inputs for the :command:`mergeCoaddDetections.py` command, which you'll run next.
+The resulting datasets are the ``deepCoadd_det`` detections and the ``deepCoadd_calexp`` calibrated coadd exposures.
 
 .. _getting-started-tutorial-merge-coadd-detections:
 
 Merging multi-band detection catalogs
 =====================================
 
-Next, use the :command:`mergeCoaddDetections.py` command to combine the individual ``HSC-R`` and ``HSC-I``-band detection catalogs:
-
-.. code-block:: bash
-
-   mergeCoaddDetections.py DATA --rerun coaddPhot --id filter=HSC-R^HSC-I
-
-This command created a ``deepCoadd_mergeDet`` dataset, which is a consistent table of sources across all filters.
+Merging the detections from the multiple bands used to produce the coadds allows later steps to use multi-band information in their processing: e.g. deblending.
+The ``mergeDetections`` sub-pipeline created a ``deepCoadd_mergeDet`` dataset, which is a consistent table of sources across all filters.
```

Comment:
This also seems a bit ambiguous about if you are supposed to run something separate. Perhaps an intro to this sections that says they are all included in the coadd_measurement subset?